cmd/harmony/main.go

```diff
@@ -329,6 +329,13 @@ func setupNodeAndRun(hc harmonyConfig) {
 		HTTPPort:    hc.HTTP.RosettaPort,
 	}
 
+	// Pares Prometheus config
+	nodeConfig.PrometheusServer = nodeconfig.PrometheusServerConfig{
+		HTTPEnabled: hc.HTTP.PrometheusEnabled,
+		HTTPIp:      hc.HTTP.IP,
```

Comment:
Shall we create a different IP field for prometheus service? E.g, user want to open prometheus to 0.0.0.0, but RPC to local 127.0.0.1?

Can we also create the IP flag for rosetta?

Reply:
make sense.  For rosetta, @Daniel-VDM can add a new variable for the IP.